Tests/Util/TestAccountFactory.swift

```diff
@@ -0,0 +1,116 @@
+//
+//  Copyright (c) 2020-2023 MobileCoin. All rights reserved.
+//
+@testable import MobileCoin
+import XCTest
+
+#if swift(>=5.5)
+
+@available(iOS 15.0, *)
+
+public struct TestAccountFactory {
+
+    enum TestAccountFactoryError: Error {
+        case unknownError(_ description: String)
+    }
+
+    let fogReportUrl: String
+    let fogReportId: String = ""
+    let fogAuthoritySpki: Data
+    let accountIndex: UInt32 = 0
+
+    public struct TestAccountConfig {
+        let name: String
+        let txData: [TokenId: [UInt64]]
```

Comment:
can we document what txData is storing ? or maybe instead of storing UInt64, store Amount objects then we're surfacing those semantics to the next person.

Reply:
Sure, I can switch this to just an array of Amounts... In fact, that gives more control over the ordering of the transactions created (the way it is now will group the individual transactions by token ID).

Reply:
done